lectures/french_rev.md

```diff
@@ -1,1031 +1,1026 @@
-<!-- #endregion -->
-
-```{code-cell} ipython3
-# figure 15
-data15 = pd.read_excel('datasets/assignat.xlsx', sheet_name='Post-95', usecols='N', skiprows=4, nrows=88, header=None).squeeze()
-
-plt.figure()
-h = plt.plot(range(2, 90), data15, '*-', linewidth=0.8)
-plt.setp(h, markersize=2)
-plt.gca().spines['top'].set_visible(False)
-plt.gca().spines['right'].set_visible(False)
-plt.text(47.5, 11.4, '17 brumaire', horizontalalignment='left', fontsize=12)
-plt.text(49.5, 14.75, '19 brumaire', horizontalalignment='left', fontsize=12)
-plt.text(15, -1, 'Vendémiaire 8', fontsize=12, horizontalalignment='center')
-plt.text(45, -1, 'Brumaire', fontsize=12, horizontalalignment='center')
-plt.text(75, -1, 'Frimaire', fontsize=12, horizontalalignment='center')
-plt.ylim([0, 25])
-plt.xticks([], [])
-plt.ylabel('Francs')
-
-plt.tight_layout()
-plt.show()
-#plt.savefig('frfinfig15.pdf', dpi=600)
-```
-
-```{code-cell} ipython3
-
-```
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+<!-- #region user_expressions=[] -->
+# Inflation During French Revolution 
+
+
+## Overview 
+
+This lecture describes some monetary and fiscal  features of the French Revolution (1789-1799)
+described by {cite}`sargent_velde1995`.
+
+To finance public expenditures and service its debts, 
+the French Revolutionaries  performed several policy experiments.
+
+The Revolutionary legislators who authored these experiments were guided by their having decided to put in place monetary-fiscal policies recommended to them  by theories that they believed.
+
+Some of those theories make contact with modern theories about monetary and fiscal policies that interest us today.
+
+* a *tax-smoothing* model like Robert Barro's {cite}`Barro1979`
+
+   * this normative (i.e., prescriptive mode) advises a government to finance temporary war-time surges in government expenditures mostly by issuing government debt; after the war to roll over whatever debt accumulated during the war, and  to increase taxes permanently by enough to finance interest payments on that post-war debt
+
+*  *unpleasant monetarist arithmetic* like that described in this quanteon lecture  {doc}`unpleasant`
+   
+    * this arithmetic governed French government debt dynamics in the decades preceding 1789 and according to leading historians set the stage for the French Revolution 
+
+* a *real bills* theory of the effects of government open market operations in which the government *backs* its issues of paper money with valuable real property or financial assets
+
+    * the Revolutionaries learned about this theory from Adam Smith's 1776 book The Wealth of Nations
+     and other contemporary sources
+
+    * It shaped how the Revolutionaries issued paper money called assignats from 1789 to 1791 
+
+* a classical **gold**  or **silver** standard
```

Comment:
```suggestion
* a classical *gold*  or *silver* standard
```

Reply:
Many thanks @mmcky,

I think this is a definition (gold standard and silver standard) so I also set "standard" as bold as well.

Please let me know your thoughts on this : )